web/client/api/WMS.js

```diff
@@ -77,7 +77,7 @@ export const searchAndPaginate = (json = {}, startPosition, maxRecords, text) =>
     const root = (json.WMS_Capabilities || json.WMT_MS_Capabilities || {}).Capability;
     const service = (json.WMS_Capabilities || json.WMT_MS_Capabilities || {}).Service;
     const onlineResource = getOnlineResource(root);
-    const SRSList = root.Layer && (root.Layer.SRS || root.Layer.CRS) || [];
+    const SRSList = root.Layer && (root.Layer.SRS?.map((srs) => srs.toUpperCase()) || root.Layer.CRS?.map((crs)=>crs.toUpperCase())) || [];
```

Comment:
Please add a unit test for this change.
Note: we could write this .map once, something like this: 
```js
(root.Layer.SRS || root.Layer.CRS)?.map((crs) => crs.toUpperCase())
```

Reply:
Or Nice one @allyoucanmap. Did not know that one

Reply:
@MuhweziDeo or probably in this case is better:
```js
const SRSList = (root.Layer && (root.Layer.SRS || root.Layer.CRS) || []).map((srs) => srs.toUpperCase());
```